### PR TITLE
Accept invitation to join the team

### PR DESCRIPTION
**Pull request by andymotagl**
## Description

Adds the functionality of creating a new player based on an invitation token.

The invitation token expires after 7 days.

Create a dummy resource to respond the future link with the invitation token generated.

### CODE REVIEW

File: app/src/service/player.js [hunk 1]
```diff
@@ -22,46 +22,41 @@ const getAll = async (actorId) => {
...
-    }
+const create = async (email, teamId, actorId, token) => {
+    if (!email || !emailTool.validate(email)) throw new BadRequest('Invalid email');
+    
+    const actorByEmail = await actorDAO.getActorByEmail(email);
+    if (actorByEmail) throw new BadRequest('A Player with email specified already exists');
+    
+    if (!token && (!actorId || !uuidTool.validate(actorId))) throw new BadRequest('Leader id invalid');
+    if (!teamId && !uuidTool.validate(teamId)) throw new BadRequest('Invalid Team id');
+
```
Comment (FernandoArevaloE) [inline app/src/service/player.js:35]: I think we need to standarize messages. We are using "... id invalid" and "Invalid ... id". This should also apply for not found messages and others

File: app/src/service/player.js [hunk 2]
```diff
@@ -22,46 +22,41 @@ const getAll = async (actorId) => {
...
-        createdBy: leader.actor_id,
-        companyId: leader.company_id
+        createdBy: team.actor_id,
+        companyId: team.company_id
     };
     const actor = await actorDAO.createActor(newActor);
-    if (!actor) {
-        throw new BadRequest('Something went wrong while creating the new player');
-    }
+    if (!actor) throw new InternalServerError('Something went wrong while creating the new player');
```
Comment (FernandoArevaloE) [inline app/src/service/player.js:47]: was it suppose that an insert sentence should work or returns an exception? Then why do we need to validate a result when running a creation?

File: app/src/service/player.js [hunk 3]
```diff
@@ -22,46 +22,41 @@ const getAll = async (actorId) => {
...
     const newPlayer = {
         actorId: actor.actor_id,
         teamId: teamId
     };
 
-    return await playerDAO.create(newPlayer);
+    const player = await playerDAO.create(newPlayer);
+    if (!player) throw new InternalServerError('Something went wrong while creating the new player');
+    return player;
+}
```
Comment (FernandoArevaloE) [inline app/src/service/player.js:56]: same above

File: app/src/service/player.js (hunk 1 above)
Comment (henrymcarvajal) [reply app/src/service/player.js:35]: English syntax is [adjective][noun]... so we should use "Invalid whatever...", "Bad wtf", etc

File: app/src/service/player.js (hunk 2 above)
Comment (henrymcarvajal) [reply app/src/service/player.js:47]: We agreed on that _after_ Andres developed this...  a simple deletion would suffice

File: app/src/service/player.js (hunk 3 above)
Comment (henrymcarvajal) [reply app/src/service/player.js:56]: Same as above ☠️

**Review by henrymcarvajal — COMMENTED** [review]
Up!

**Review by FernandoArevaloE — APPROVED** [review]

**Pull request merged**